packages/next-templates/src/app/globals.css

```diff
@@ -67,3 +67,13 @@ https://www.figma.com/file/gX3fq7k0uUKC45AJbuItDv/Local---Voorbeeld---Bibliothee
   --utrecht-heading-2-font-size: 20px;
   --utrecht-spotlight-section-padding-block-end: 24px;
 }
+
```

Comment:
kan hier een comment zetten dat de radio button nog gestyled moet worden in het thema